### PR TITLE
Keyboard shortcuts 

### LINKED ISSUE #2052: ‘⌘,’ should bring up NNW Settings
I semi-frequently find myself hitting ‘⌘,’ (CMD-,) with Magic Keyboard to bring up the Settings (Preferences?) pane In NetNewsWire but it doesn’t respond. Similarly, ‘⌘W’ (CMD-W) should dismiss it (as well as esc).

### PR DESCRIPTION
**Pull request by rizwankce**
- toggle side bar on timeline - cmd+ctrl+S
- toggle readers view on article - cmd+shift+R
- go to settings on global - cmd+,

Fixes #2052 , #2029 , #2008

### CODE REVIEW

File: iOS/MasterTimeline/MasterTimelineViewController.swift [hunk 1]
```diff
@@ -156,7 +156,11 @@ class MasterTimelineViewController: UITableViewController, UndoableCommandRunner
 	@objc func showFeedInspector(_ sender: UITapGestureRecognizer) {
 		coordinator.showFeedInspector()
 	}
-	
+
+	@objc func toggleSidebar(_ sender: Any?) {
+		coordinator.toggleSidebar()
+	}
+
```
Comment (vincode-io) [inline iOS/MasterTimeline/MasterTimelineViewController.swift:163]: I would have thought that this needed to go into RootSplitViewController to work globally.  Does toggling the sidebar work okay when focus is on the Timeline or Article?

Comment (rizwankce) [reply iOS/MasterTimeline/MasterTimelineViewController.swift:163]: Let me double check and get back to you on the behaviour.

Comment (rizwankce) [reply iOS/MasterTimeline/MasterTimelineViewController.swift:163]: Double checked. its working when focusing on timeline or article.

Comment (rizwankce) [reply iOS/MasterTimeline/MasterTimelineViewController.swift]: Moved the `toggleSidebar` to `RootSpiltViewController` and verified it on iPad Pro. Working fine on timeline and article. @vincode-io

**Pull request merged**